docs/docs/recoil-sync/api/RecoilURLSyncTransit.md

```diff
@@ -21,51 +21,55 @@ Transit encoding is not as terse or readable as just using [JSON](/docs/recoil-s
 ## Custom Classes
 
 Handlers for custom user classes can be defined with the `handlers` prop:
-```jsx
+
+```tsx
 type TransitHandler<T, S> = {
-  tag: string,
-  class: Class<T>,
-  write: T => S,
-  read: S => T,
+  tag: string;
+  class: Class<T>;
+  write: (T) => S;
+  read: (S) => T;
 };
 ```
 
 ### Example
-```jsx
+
+```tsx
 class ViewState {
   active: boolean;
   pos: [number, number];
   constructor(active, pos) {
     this.active = active;
     this.pos = pos;
   }
-  ...
-};
-const viewStateChecker = custom(x => x instanceof ViewState ? x : null);
+  // ...
```

Comment:
commented this out so prettier can recognize the syntax as valid and do it's thing